pygeoapi/provider/postgresql.py

```diff
@@ -149,6 +152,8 @@ def __init__(self, provider_def):
         self.table = provider_def['table']
         self.id_field = provider_def['id_field']
         self.conn_dic = provider_def['data']
+        self.geom = provider_def.get('geom_field') \
+            if provider_def.get('geom_field') else "geom"
```

Comment:
Why not `self.geom = provider_def.get('geom_field', 'geom')` ?

Reply:
@alpha-beta-soup  Implemented recommendation, thank you